### PR TITLE
Update klint plugins (DEV)

### PR DESCRIPTION
**Pull request by mtwalli**
Update klint plugins to enable training commas, Please not changes in here are from klint formatting

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,8 +18,8 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath "com.google.protobuf:protobuf-gradle-plugin:$protobufVersion"
         classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$navVersion"
-        classpath "org.jlleitschuh.gradle:ktlint-gradle:9.2.1"
-        classpath "io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.9.1"
+        classpath "org.jlleitschuh.gradle:ktlint-gradle:9.4.1"
```
Comment (d4rken) [inline build.gradle:21]: Why does "Allowing trailing commas", require 127 files to be changed?

A lot of the reformatting is unnecessary or worse imho, and the potential for merge conflicts is huge.

**Review by d4rken — CHANGES_REQUESTED** [review]
The PR should be about the linter allowing trailing commas. Just updating for the sake of updating is not desired if it fixes no issues and reformats 127 files. Linting rules change almost monthly....

File: build.gradle (hunk 1 above)
Comment (d4rken) [reply build.gradle:21]: I appreciate the effort :heart:  but would like to reduce this PR to "just allowing trailing commas".

Comment (mtwalli) [reply build.gradle:21]: Well I looked for a way to enable only the training commas did not find a rule that was preventing it in our config file.
After some Googling I found out that it is only possible by upgrading the plugin

Comment (d4rken) [reply build.gradle:21]: I don't want to decide this. We should check with the rest of the team.

Comment (mtwalli) [reply build.gradle:21]: Fair enough

**Comment by BMItter** [comment]
I'd be ok with this change.  also like the formatting

File: build.gradle (hunk 1 above)
Comment (chris-cwa) [reply build.gradle:21]: i also fear a lot of merge conflicts. is this a reasonable price for trailing commas?

**Comment by mtwalli** [comment]
Closing it, till we agree on a way of doing it

**Pull request closed without merging**